pkgs/top-level/all-packages.nix

```diff
@@ -28339,6 +28339,8 @@ with pkgs;
 
   savepagenow = callPackage ../tools/misc/savepagenow { };
 
+  savvycan = libsForQt5.callPackage ../applications/networking/sniffers/savvycan {};
+
```

Comment:
This is not in the right section

Reply:
what do you mean by `DATA`?

Reply:
Ok I looked into it and placed the line in the application section. The problem is that there is no correct ordering so I put it in the best available place.

Reply:
That's fine, the ordering is a mess, hopefully https://github.com/NixOS/rfcs/pull/140 will fix it